README.md

```diff
@@ -6,20 +6,23 @@
 
 # nrich
 
-Nrich is a Java library developed at CROZ whose purpose is to make development of applications on JVM a little easier. It contains modules that were found useful on projects and as such were extracted
-to a common library. It is based on Spring Framework it also provides Spring Boot starters for most of the modules to make configuration easier.
+Nrich is a Java library developed at CROZ whose purpose is to make development of applications on the JVM a little
+easier.
+It was created by combining modules that were found useful on multiple projects into a common library.
+The library is based on the Spring Framework, and as such provides Spring Boot starters for most of the modules
+to make the configuration easier.
 
 Most modules are composed of multiple submodules:
```

Comment:
Not sure about this sentence. It sounds like there is a parent module that is composed of three submodules but that is not the case here. In reality, we have three modules that are related to one group (e.g. validation).

Reply:
I agree. I will change everywhere where I made this mistake.